core/index.html

```diff
@@ -896,7 +896,7 @@ <h2><dfn>Link</dfn></h2>
 </figure>
 
       <figure><figcaption>
-        Alternatively, if additional metadata is required (such as the MIME
+        Alternatively, if additional metadata is required (such as the
         content type of the referenced resource) a <a>Link</a> can be used:
```

Comment:
presumably this should also be media type?

Reply:
Content type is often associated with the HTTP `Content-Type` header field, which specifies the media type of an entire HTTP response or request body. Updated along with another case: https://github.com/w3c/activitystreams/pull/639/commits/12c4cafaca0c969ddc479a56fde7b586e1e4dfc9